scripts/utils/addresses.ts

```diff
@@ -0,0 +1,32 @@
+import { NetworkType } from '@holographxyz/networks';
+
+type TokenSymbol = 'hETH' | 'hBNB' | 'hAVAX' | 'hMATIC' | 'hMNT';
+type HTokenAddresses = {
+  [network in NetworkType]: {
+    [token in TokenSymbol]?: string;
+  };
+};
+
+export const tokenAddresses: HTokenAddresses = {
```

Comment:
Since this is moved to a file called addresses.ts, I think it makes sense to update the variable to be specific since there could be other contract addresses in this file. Maybe `hTokenAddresses`